### PR TITLE
My enumerable

### PR DESCRIPTION
**Pull request by Rustamxon7**
### Create your own Enumerable

- All required files are installed. 💯
- MyList class is created. 💯

- MyEnumerable implements all required and optional methods:
> #all? #any? #filter? #max #min #sort 💯

Feel free to review 🧑🏻‍💻 our PR and let us know if there are any changes to do. 🤓

### CODE REVIEW

File: mylist.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+class MyList
```
Comment (Mihndim2020) [inline mylist.rb:1]: - [ ] Please you will have to import `myenumerable.rb` file in to `mylist.rb` to be able to access the enumerable methods, please kindly fix that.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @Rustamxon7, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: mylist.rb (hunk 1 above)
Comment (Donard97) [reply mylist.rb:3]: Thank you for telling us. We imported myenumerable.rb file in to mylist.rb.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**